Packs/Bitbucket/Integrations/Bitbucket/Bitbucket_description.md

```diff
@@ -1,14 +1,18 @@
 ## Bitbucket Integration Help
 
 * To create an app password to Bitbucket
-  1. Go to your account > **Settings** > **App Passwords**.
-  2. Click **Create app password**.
-  3. Give the app password a name related to the application that will use the password.
-  4. Select the specific access and permissions you want this application password to have.
-  5. Copy the generated password and either record or paste it into the application you want to give access. The password is only displayed this one time.
-  6. Click [here](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#authentication) for more information.
+  1. Select your avatar (The icon that represent your profile and settings) from the navigation bar at the top of the screen.
```

Comment:
Don't think we need this. 
1. Go to your account -> settings -> Personal Settings
2. Select App password